src/routes/Dashboard/CardList/Card.tsx

```diff
@@ -1,9 +1,8 @@
-import { useMemo } from 'react'
-import { IncreaseIcon, DecreaseIcon } from 'assets/svgs'
-
-import { transformNum } from './utils'
+import { useEffect, useMemo } from 'react'
+import { IncreaseIcon, DecreaseIcon, MinusIcon } from 'assets/svgs'
 
 import styles from './cardList.module.scss'
+import { transformNum } from 'utils/transformNum'
```

Comment:
```suggestion
import { useEffect, useMemo } from 'react'

import { transformNum } from 'utils/transformNum'

import { IncreaseIcon, DecreaseIcon, MinusIcon } from 'assets/svgs'
import styles from './cardList.module.scss'
```
import 순서 고려해주는게좋아보입니다